validations/users.js

```diff
@@ -0,0 +1,44 @@
+function validateUsername(username) {
+    if (!username) {
+        throw new Error('Error. There was no Username provided.');
+    }
+}
+
+function validatePassword(password) {
+    if (!password) {
+        throw new Error('Error. There was no password provided.');
+    }
+}
+
+function validateLoginSession(userLoginSession) {
+    if (!userLoginSession) {
+        throw new Error('Error. There was no users session data provided.');
+    }
+}
+
+function validateKey(key) {
+    if (!key) {
+        throw new Error("Error. You didn't provide any key to update.");
+    }
+}
+
+function validateValue(value) {
+    if (value === undefined || value === 'undefined') {
```

Comment:
Why do we need to check if `value === 'undefined'` here?

Reply:
I don't really remember. That doesn't look logical now.  Fixed.